parser-library/nt-headers.h

```diff
@@ -38,6 +38,7 @@ const std::uint16_t NUM_DIR_ENTRIES = 16;
 const std::uint16_t NT_OPTIONAL_32_MAGIC = 0x10B;
 const std::uint16_t NT_OPTIONAL_64_MAGIC = 0x20B;
 const std::uint16_t NT_SHORT_NAME_LEN = 8;
+const std::uint16_t SYMTAB_RECORD_LEN = 18;
```

Comment:
I think you can get rid of all the `std::` for these integer types, and just include `<cstdint>`. Most of the rest of the code doesn't use `std::` explicitly.